### PR TITLE
Update environment and travis for ctapipe_master branch

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

**Comment by rlopezcoto** [comment]
?
Is this a duplicate of #163?

**Comment by vuillaut** [comment]
> ?
> Is this a duplicate of #163?

Nop.
#163 will be merged in the master branch of lstchain which fixed ctapipe=0.7

This will be merged in a separate development branch, called `ctapipe_master` used to make sure lstchain does not go too far from the master version of ctapipe.
Please see [the slides](
https://indico.cta-observatory.org/event/2379/contributions/20975/attachments/15994/20604/20190730_lstchain_branches_workflow_vuillaume.pdf) for a better understanding or reminder of the workflow we discussed.

**Comment by rlopezcoto** [comment]
Yeah, sorry I did not realize it was committed to the `ctapipe_master` branch, sorry

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +35,6 @@ install:
     - conda env create --name cta --file environment.yml
     - conda activate cta
     - pip install https://github.com/cta-observatory/ctapipe/archive/$CTAPIPE_VERSION.tar.gz
-    - pip install https://github.com/cta-sst-1m/protozfitsreader/archive/$PROTOZFITS_VERSION.tar.gz
     - pip install https://github.com/cta-observatory/ctapipe_io_lst/archive/$CTAPIPE_IO_LST_VERSION.tar.gz
```
Comment (rlopezcoto) [inline .travis.yml]: can we re-add here the PROTOZFITS reader as well?

Comment (vuillaut) [reply .travis.yml]: done

**Comment by rlopezcoto** [comment]
Thanks!

**Pull request merged**